assessment.rb

```diff
@@ -1,12 +1,32 @@
 require 'csv'
 # Assessment answers to:
+rails g model Licensee surname:string given_name:string gender:string height:integer weight:integer dob:date,eye_color:string hair_color:string
 
 # question 2:
+caren = Licensee.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height:63, wieght:132, dob:1943-09-26, eye_color:'Brown', hair_color:'Black'
 
 # question 3:
+rails g model CreateLicensees
```

Comment:
You already created the model and migration when you ran `rails g model Licensee`; did you mean to create another model/migration?